### PR TITLE
Add ddwrt docs for new type and monitored_condition conf

### PR DESCRIPTION
**Pull request by Extarys**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here> https://github.com/home-assistant/home-assistant/pull/22743

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by Extarys** [comment]
Huh, I'm not quite sure I understand the error given by `netlify` as I didn't touch the line 41.

File: source/_components/ddwrt.markdown [hunk 1]
```diff
@@ -50,6 +54,32 @@ verify_ssl:
...
+  type: list
+  keys:
+    interface:
+      description: Display the interface name (w0, w1, w2)
+    tx_rate:
+      description: The connection speed of the downlink, in Mbps
+    rx_rate:
+      description: The connection speed of the uplink, in Mbps
+    info:
+      description: Connection type: Legacy, HT20, VHT80PS...
```
Comment (klaasnicolaas) [inline source/_components/ddwrt.markdown]:
```suggestion
      description: "Connection type: `Legacy`, `HT20`, `VHT80PS`."
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ddwrt.markdown [hunk 2]
```diff
@@ -25,6 +25,10 @@ device_tracker:
     host: ROUTER_IP_ADDRESS
     username: YOUR_ADMIN_USERNAME
     password: YOUR_ADMIN_PASSWORD
+    type: lan
```
Comment (klaasnicolaas) [inline source/_components/ddwrt.markdown]: Please remove

File: source/_components/ddwrt.markdown (hunk 1 above)
Comment (Extarys) [reply source/_components/ddwrt.markdown]: Oh ok thanks I asked about this but we weren't sure, I'll update now

File: source/_components/ddwrt.markdown [hunk 3]
```diff
@@ -25,6 +25,10 @@ device_tracker:
...
+    monitored_conditions:
```
Comment (klaasnicolaas) [inline source/_components/ddwrt.markdown]: Please remove

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ddwrt.markdown (hunk 1 above)
Comment (Extarys) [reply source/_components/ddwrt.markdown]: I don't know the list of possible values, there are quite a lot.

File: source/_components/ddwrt.markdown (hunk 3 above)
Comment (Extarys) [reply source/_components/ddwrt.markdown]: ok

File: source/_components/ddwrt.markdown (hunk 2 above)
Comment (Extarys) [reply source/_components/ddwrt.markdown]: ok

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**